### ISSUE TITLE
Reason for node.js >= 5?

### ISSUE THREAD

**Issue by michaelswe** [futurice/pepperoni-app-kit#74]
What is the reason for needing Node >=5 as stated in the setup guide? I'd rather stick with LTS version for production (4.4.7 for the moment). Everything seems to work fine with 4.4.7.

**Comment by jhabdas** [comment]
Every even version of node is an LTS. Have you considered upgrading, @michaelswe?

**Comment by jevakallio** [comment]
@michaelswe Node >= 5 is what we developed and tested Pepperoni on, so we put that version as the minimum to avoid unexpected behaviour. I tested briefly with node 4, and everything worked all right, so that should be fine.

The node version only matters during build and development time, though - the deployed app code runs in JavaScriptCore on the device, so "for production", as you say, the node version does not matter.

I won't downgrade the minimum requirement in the documentation, as more likely we'll be hitching it up to 6 sooner or later.
